src/PhpPact/Consumer/Matcher/Formatters/PluginFormatter.php

```diff
@@ -51,7 +48,7 @@ public function format(MatcherInterface $matcher): string
         throw new MatcherNotSupportedException(sprintf("Matcher '%s' is not supported by plugin", $matcher->getType()));
```

Comment:
I think I will replace this exception by `BadMethodCallException('Method not allowed')` once I removed all of the `if` conditions.

Reply:
I was wrong. This method is still required for rules for EachValue/EachKey matchers.